tests/test_base.py

```diff
@@ -46,6 +50,35 @@ def test_base_delete_connection_error():
         j = n.job.deregister_job("example")
 
 
+@mock.patch("nomad.api.base.requests.Session")
+def test_base_raise_exception_not_requests_response_object(mock_requests):
+    mock_requests().delete.side_effect = [requests.RequestException()]
+
+    try:
+        n = nomad.Nomad(
+            host="162.16.10.102",
```

Comment:
Out of curiosity, is this an actual host of importance or just a dummy value?

Reply:
Completely arbitrary dummy value!